package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "dotenv-flow": "^4.0.1",
+    "react-icons": "^5.0.1",
```

Comment:
Would prefer not to add another icon pack/dependency just for one icon. We already have react bootstrap icons, let's use [one of their icons.](https://icons.getbootstrap.com/?q=pen)